CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [develop]
 
 ### Added
+- New `ListBox` UI element to display multiple selectable items
```

Comment:
`New` is redundant. It's implicitly new as it's in the `Added` section. Just writing `` `ListBox` UI element`` is sufficient.

Also add an entry for the demo, e.g.

```
### Added
- Demo section in playground with separate `SettingsPanel`s for subtitle and audio track selection via `ListBox`
```